<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sample_4_2Class</class>
 <widget class="QWidget" name="sample_4_2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sample_4_2</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelStr1">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>str1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxStr1">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>g:\Qt5Book\QT5.9Study</string>
       </property>
       <item>
        <property name="text">
         <string>g:\Qt5Book\QT5.9Study</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>D:\sourcetree\hitPaw-video-download</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelStr2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>str2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxStr2">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>\</string>
       </property>
       <item>
        <property name="text">
         <string>\</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>字符串</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnAppend">
          <property name="text">
           <string>Append</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnPrepend">
          <property name="text">
           <string>Prepend</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnToUpper">
          <property name="text">
           <string>toUpper</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnToLower">
          <property name="text">
           <string>toLower</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnLeft">
          <property name="text">
           <string>left</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnRight">
          <property name="text">
           <string>right</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="btnSection">
          <property name="text">
           <string>section</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="btnSimple">
          <property name="text">
           <string>simplified</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="btnTrimmed">
          <property name="text">
           <string>trimmed</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>数字</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="btnSize">
          <property name="text">
           <string>size</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="btnLastIndexOf">
          <property name="text">
           <string>lastIndexOf</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnIndexOf">
          <property name="text">
           <string>indexOf</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnCount">
          <property name="text">
           <string>count</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>逻辑判断</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnEnds">
          <property name="text">
           <string>endsWith</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnIsNull">
          <property name="text">
           <string>startsWith</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnStarts">
          <property name="text">
           <string>contains</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="btnContains">
          <property name="text">
           <string>isNull</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnIsEmpty">
          <property name="text">
           <string>isEmpty</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelResult">
       <property name="minimumSize">
        <size>
         <width>39</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editResult">
       <property name="text">
        <string>g:\Qt5Book\QT5.9Study</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labSpin">
       <property name="text">
        <string>LabSpin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimumSize">
        <size>
         <width>132</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sample_4_2.qrc"/>
 </resources>
 <connections/>
</ui>
